src/routes/areas/deleteArea.test.ts

```diff
@@ -0,0 +1,47 @@
+import { FastifyInstance, RouteOptions } from 'fastify';
+import createServer from '../../server';
+import deleteArea from './deleteArea';
+import AreaModel from '../../models/area';
+
+describe('DELETE /area', () => {
+  let server: FastifyInstance;
+  beforeAll(() => {
+    server = createServer({ authRoute: [deleteArea] as RouteOptions[] });
+  });
+
+  it('returns a 204 status code when deletes an area correctly ', async () => {
+    const areaId = 2;
+
+    await AreaModel.query().insert({
+      name: 'DELETE',
+      CreatedBy: 'ADMIN_TEST',
+      ModifiedBy: 'ADMIN_TEST',
+      CreatedDate: new Date(),
+      ModifiedDate: new Date(),
+    });
+
+    const response = await server.inject({
+      method: 'DELETE',
+      url: `/area/${areaId}`,
+      payload: {
+        name: 'deleteTest',
```

Comment:
❗If we delete this area, why are we passing a payload? 🤔

Reply:
Ooooh, apparently, when this endpoint was created, the Body was added to the **DeleteAreaRoute** but is not used in the handler. Please modify the test and the **deleteArea** file to not request the body. @leandro0day